src/url-helper.ts

```diff
@@ -12,7 +12,8 @@ export function getFetchUrl(settings: IGitSourceSettings): string {
   const encodedOwner = encodeURIComponent(settings.repositoryOwner)
   const encodedName = encodeURIComponent(settings.repositoryName)
   if (settings.sshKey) {
-    return `git@${serviceUrl.hostname}:${encodedOwner}/${encodedName}.git`
+    let user = settings.sshUser.length > 0 ? settings.sshUser : 'git'
```

Comment:
Not sure if we care too much about `let` vs `const` in this repo, so feel free to ignore

```suggestion
    const user = settings.sshUser.length > 0 ? settings.sshUser : 'git'
```